src/rum/core.cljs

```diff
@@ -98,7 +103,14 @@
            (fn []
              (this-as this
                (let [state @(state this)]
-                 (clj->js (transduce (map #(% state)) merge {} child-context))))))}
+                 (clj->js (transduce (map #(% state)) merge {} child-context))))))
+         :contextTypes
+         (when-not (empty? context-types)
+           (clj->js (apply merge context-types)))
+         :childContextTypes
+         (when-not (empty? child-context-types)
+           (clj->js (apply merge child-context-types)))}
+
```

Comment:
Using a merge here that warns (or throws even) on conflicts would be good I think. This way using 3rd party mixins is "safe" and free from surprises.